.github/workflows/sync-examples.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v3
       - name: Run GitHub File Sync
-        # Can update to v1 when https://github.com/BetaHuhn/repo-file-sync-action/issues/168 is resolved
-        uses: BetaHuhn/repo-file-sync-action@v1.16.5
+        uses: BetaHuhn/repo-file-sync-action@v1
```

Comment:
https://github.com/BetaHuhn/repo-file-sync-action/issues/168 is still open, I think this should wait.

Reply:
Yeah I just wanted to test it

Reply:
Will likely fail. :sweat_smile: We could ping the author

Reply:
Reverted update, but included the `ORIGINAL_MESSAGE: true` still.

It includes the original commit msg, instead of that `update LICENSE` all the time :joy: